docs/topics/proc_deploying-the-caching-booster-using-the-oc-cli-client.adoc

```diff
@@ -0,0 +1,91 @@
+[#deploying-the-caching-booster-using-the-oc-cli-client_{context}]
+= Deploying the {name-mission-caching} Booster using the `oc` CLI Client
+
+.Prerequisites
+
+* The booster application created using {parameter-deployment}.
+ifndef::parameter-openshiftlocal[For more information, see xref:deploying-the-booster-using-launcher-oso_{context}[].]
```

Comment:
Should there be a third if for the OCP case?

Reply:
@Ladicek no, since in the instructions for OCP, we defer back to the other two sections.

E.g. https://appdev.openshift.io/docs/nodejs-runtime.html#mission-rest-http-nodejs (3.1.3)